src/admin-views/help-calendar.php

```diff
@@ -2,12 +2,14 @@
 
 use \Tribe\Admin\Help_Page;
 
-$faqs              = tribe( Tribe__Admin__Help_Page::class )->get_calendar_faqs();
-$extensions        = tribe( Tribe__Admin__Help_Page::class )->get_calendar_extensions();
-$calendar_products = tribe( Tribe__Admin__Help_Page::class )->get_calendar_products();
+$faqs                    = tribe( Tribe__Admin__Help_Page::class )->get_calendar_faqs();
+$extensions              = tribe( Tribe__Admin__Help_Page::class )->get_calendar_extensions();
+$calendar_products       = tribe( Tribe__Admin__Help_Page::class )->get_calendar_products();
+$is_tec_events_help_page = tribe( Tribe__Admin__Help_Page::class )->is_tec_events_help_page();
+$tec_events_style        = $is_tec_events_help_page ? 'block' : 'none';
 
 ?>
-<div id="tribe-calendar">
+<div id="tribe-calendar" style="display: <?php echo esc_attr( $tec_events_style ); ?>;">
```

Comment:
We have classes to do this so we don't have to rely on inline styles. See https://github.com/the-events-calendar/tribe-common/blob/master/src/resources/postcss/a11y/_visibility.pcss